locales/en.lua

```diff
@@ -33,6 +33,7 @@ local Translations = {
             vehicles = "Available Vehicles",
             depot = "%{value} [ $%{value2} ]",
             garage = "%{value} [ %{value2} ]",
+            unavailable_vehicle_model = "%{vehicle} is unavailable resource",
```

Comment:
Could you please change this to:
`unavailable_vehicle_model = "%{vehicle} | Vehicle Unavailable",`

Reply:
Sorry for the late reply. I will make changes to the PR for your edit.